### PR TITLE
init sponsors

### PR DESCRIPTION
**Pull request by MasonEgger**

### CODE REVIEW

File: pelicanconf.py [hunk 1]
```diff
@@ -111,7 +113,8 @@
 }
 
 JINJA_FILTERS = {
-    "sidebar": sidebar
+    "sidebar": sidebar,
+    "md": md,
```
Comment (nimbinatus) [inline pelicanconf.py]: Getting an error that md is not defined when building locally (did you do a `poetry run make clean` before `poetry run make devserver`?). What's this addition for?

Comment (nimbinatus) [reply pelicanconf.py]: If this is to make markdown work in html, there's a different way to do it, and I'd recommend that way with all of the changes I'm suggesting:
```
from markdown.extensions.md_in_html import MarkdownInHtmlExtension
...
MARKDOWN = {
    [
        TableExtension(),
        MarkdownInHtmlExtension()
    ]
}
```
And remove this line.

File: content/sponsors/sponsors.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<div class="container">
+  <h3>Platinum</h3>
+    <div class="row mb-4">
+      <div class="col-4">
+        <a href="">
+        <img src="">
+        </a>
+      </div>
+    </div>
+-->
```
Comment (nimbinatus) [inline content/sponsors/sponsors.md]:
```suggestion
```

File: content/sponsors/sponsors.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+#### Door Prize Sponsors
+<div class="row mb-4">
+  <div class="col-3">
+    <a href="https://nostarch.com/">
+    <img src="/theme/img/sponsors/no-starch.png" alt="No Starch Press logo">
+    </a>
+  </div>
+
+</div>
```
Comment (nimbinatus) [inline content/sponsors/sponsors.md]:
```suggestion
<div class="row mb-4" markdown="1">
<div class="col-3" markdown="1">

[![No Starch Press logo](/theme/img/sponsors/no-starch.png)](https://nostarch.com/)

</div>
</div>
```

File: content/sponsors/sponsors.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    <div class="col-3">
+      <a href="https://2023.pythonwebconf.com/">
+      <img src="/theme/img/sponsors/pwc23.png" alt="Python Web Conf Logo">
+      </a>
+    </div>
+</div>
+
+</div>
+
+
```
Comment (nimbinatus) [inline content/sponsors/sponsors.md]:
```suggestion
<div class="row mb-4" markdown="1">
<div class="col-3" markdown="1">

[![Python Web Conf Logo](/theme/img/sponsors/pwc23.png)](https://2023.pythonwebconf.com/)

</div>
</div>

```

**Review by nimbinatus — CHANGES_REQUESTED** [review]

File: pelicanconf.py (hunk 1 above)
Comment (MasonEgger) [reply pelicanconf.py]: Oh that was something I was fiddling with but removed. I'll remove it

Comment (MasonEgger) [reply pelicanconf.py]: Commit changed and removed.

Comment (nimbinatus) [reply pelicanconf.py]: You'll still need to add the stuff I mentioned, though, to make that `markdown=1` part work on the other pages.

Comment (MasonEgger) [reply pelicanconf.py]: Ok. How about now?

**Review by nimbinatus — APPROVED** [review]
LGTM!

**Pull request merged**